src/ultimate_notion/rich_text.py

```diff
@@ -286,11 +286,16 @@ def text(
     strikethrough: bool = False,
     code: bool = False,
     underline: bool = False,
-    color: Color | None = None,
+    color: Color | BGColor | None = None,
     href: str | None = None,
 ) -> Text:
     """Create a rich text Text object from a normal string with formatting.
 
+    !!! note
+
+        With the `color` parameter the font color or the background color can be set.
```

Comment:
You may want to add this to https://ultimate-notion.com/latest/features/#notion-api-limitations